sequencer/src/block.rs

```diff
@@ -210,7 +210,9 @@ impl BlockHeader for Header {
             mut fee_merkle_tree,
             ..
         } = parent_state.clone();
-        block_merkle_tree.push(parent_header.commit()).unwrap();
+        if let Err(e) = block_merkle_tree.push(parent_header.commit()) {
+            dbg!(e);
```

Comment:
I would prefer to panic! here for now, since it shouldn't happen in any of our current tests and if it does happen the state is incorrect. Eventually we will replace the panic with fetching the missing frontier

Reply:
Ok. I'll put it back. I'm currently trying to gain insight on the failing tests.

Reply:
Rereading your comment... in our current tests we are panicking here with the error: `"Given part of merkle tree is not in memory"`